.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,58 @@
+name: System Tests
+
+on:
+  push:
+    branches:
+      - '.+-system-tests'
+#  schedule:
+#
+#    # * is a special character in YAML so you have to quote this string
+#    # Run the system tests every 3 hours
+#    - cron:  '0 */3 * * *'
+
+jobs:
+  run-system-tests-ci:
+    name: Run System Tests
+    runs-on: ubuntu-latest
+
+      # let's not run this on every fork, change to your fork when developing
+      if: github.repository == 'mlrun/mlrun'
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install automation scripts dependencies and add mlrun to dev packages
+      run: pip install -r automation/requirements.txt && python setup.py develop
+    - name: Set MLRUN_GIT_HASH env var
+      run: echo "::set-env name=MLRUN_GIT_HASH::$(git rev-parse --short $GITHUB_SHA)"
+    - name: Set MLRUN_UI_GIT_HASH env var
+      run: |
+        echo "::set-env name=MLRUN_UI_GIT_HASH::$( \
+          git clone https://github.com/mlrun/ui.git mlrun-ui 2> /dev/null && \
+          cd mlrun-ui && \
+          git rev-parse --short HEAD && \
+          cd .. && \
+          rm -rf mlrun-ui)"
+    - name: Install curl and jq
+      run: sudo apt-get install curl jq
+    - name: Set LATEST_VERSION env var
+      run: echo "::set-env name=LATEST_VERSION::$(curl -sf https://pypi.org/pypi/mlrun/json | jq -r '.info.version')"
+    - name: Run System Test CI Script
+      run: |
+        python automation/system_test/run.py run \
+          "$LATEST_VERSION-$MLRUN_GIT_HASH" \
+          ${{ secrets.SYSTEM_TEST_DATA_CLUSTER_IP }} \
+          ${{ secrets.SYSTEM_TEST_DATA_CLUSTER_SSH_PASSWORD }} \
+          ${{ secrets.SYSTEM_TEST_APP_CLUSTER_SSH_PASSWORD }} \
+          ${{ secrets.SYSTEM_TEST_GITHUB_ACCESS_KEY }} \
+          ${{ secrets.SYSTEM_TEST_MLRUN_DB_PATH }} \
+          ${{ secrets.SYSTEM_TEST_V3IO_API }} \
+          ${{ secrets.SYSTEM_TEST_V3IO_USERNAME }} \
+          ${{ secrets.SYSTEM_TEST_V3IO_ACCESS_KEY }} \
+          ${{ secrets.SYSTEM_TEST_V3IO_PASSWORD }} \
+          --mlrun-repo ghcr.io/mlrun \
```

Comment:
Please separate this to 2 different arguments:
`--mlrun-registry` and `--mlrun-repo`

Reply:
Since the context here involves both Github and docker probably better to name it `--mlrun-docker-repo`